model/bindings.go

```diff
@@ -0,0 +1,122 @@
+package model
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	yaml "github.com/cloudfoundry-incubator/candiedyaml"
+	"github.com/docker/libcompose/config"
+	"github.com/docker/libcompose/utils"
+	"github.com/rancher/rancher-compose/preprocess"
+	"github.com/rancher/rancher-compose/rancher"
+	"io/ioutil"
+	"os"
+	"reflect"
+)
+
+type mapLabel map[string]string
+
+//BindingProperty holds bindings
+type BindingProperty struct {
+	Binding
+}
+
+//Binding holds the fields for Binding
+type Binding struct {
+	Scale  int               `json:"scale"`
+	Labels map[string]string `json:"labels"`
+	Ports  []string          `json:"ports"`
+}
+
+//CreateBindingsRancher creates bindings property for rancher-compose.yml
+func CreateBindingsRancher(relativePath string) map[string]BindingProperty {
+	var rawConfigRancher config.RawServiceMap
+	var rancherComposeMap map[string]rancher.RancherConfig
+	var bindingsMap map[string]BindingProperty
+
+	var configMap map[string]config.ServiceConfigV1
+	var rawConfigDocker config.RawServiceMap
+
+	bindingsMap = make(map[string]BindingProperty)
+
+	rancherFile := relativePath + "/rancher-compose.yml"
+
+	dockerFile := relativePath + "/docker-compose.yml"
+	fileExists, err := os.Stat(dockerFile)
+	_ = fileExists
+	if err == nil {
+		yamlContent, err := ioutil.ReadFile(dockerFile)
+		if err != nil {
+			log.Errorf("Error in opening file : %v\n", err)
+		} else {
+			err = yaml.Unmarshal(yamlContent, &rawConfigDocker)
+			if err != nil {
+				log.Debugf("Error during Unmarshal for file %s : %v\n", dockerFile, err)
+			} else {
+				rawConfigDocker, err := preprocess.PreprocessServiceMap(rawConfigDocker)
+				utils.Convert(rawConfigDocker, &configMap)
+				_ = rawConfigDocker
+				if err != nil {
+					log.Debugf("Error")
+				}
+			}
+		}
+	}
+
+	rancherYaml, err := ioutil.ReadFile(rancherFile)
+	if err != nil {
+		log.Errorf("Error in opening file : %v\n", err)
+		return nil
+	}
+
```

Comment:
Oh I see, I might be neglecting those templates that only have the rancher-compose files. I'll change that